content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown

```diff
@@ -0,0 +1,281 @@
+---
+title: roxygen2 7.0.0
+author: Hadley Wickham
+date: '2019-11-05'
+slug: roxygen2-7-0-0
+categories:
+  - package
+tags:
+  - r-lib
+  - devtools
+  - roxygen2
+photo:
+  url: https://unsplash.com/photos/8XddFc6NkBY
+  author: Art Lasovsky
+---
+
+We're exceedingly happy to announce the release of [roxygen2 7.0.0](https://roxygen2.r-lib.org). roxygen2 allows you to write specially formatted R comments that generate R documentation files (`man/*.Rd`) and a `NAMESPACE` file. roxygen2 is used by over 8,000 CRAN packages.
+
+Install the latest version of roxygen2 with:
+
+```{r, eval = FALSE}
+install.packages("roxygen2")
+```
+
+This a huge release containing [many minor improvements and bug fixes](https://roxygen2.r-lib.org/news/index.html#roxygen2-7-0-0). This blog post focusses on seven major improvements:
+
+* roxygen2 is no longer (ironically!) the worst documented package. It has 
+  fresh new website, <https://roxygen2.r-lib.org>, and the vignettes have 
+  all been updated.
+
+* We've made a number of tweaks to the rendering of `.Rd`. Most importantly
+  you no longer needed to escape `%` in markdown, and functions with many
+  arguments are displayed with one argument per line. We've also removed a few 
+  old features that are no longer supported.
+
+* You can create tables and headings from markdown.
+
+* You can share text and code between your README/vignettes, and your 
+  documentation with the new `@includeRmd` tag.
```

Comment:
If you're saying between your (1) README/vignettes and (2) documentation, then you should axe the comma.